torch/_torch_docs.py

```diff
@@ -9581,9 +9581,10 @@ def merge_dicts(*dicts):
 
 Example::
 
-    >>> torch.empty((2,3), dtype=torch.int64)
+    >>> a=torch.empty((2,3), dtype=torch.int64, device = 'cuda')
+    >>> torch.empty_like(a)
```

Comment:
This example doesn't look correct because `torch.empty_like(a)` is expected to be a `torch.int64` tensor but the printed tensor here is a floating point tensor. Also the print of the device looks incorrect.
I would suggest using the latest build and trying this on the python console and printing the actual result:
```
a=torch.empty((2,3), dtype=torch.int32, device='cuda')
torch.empty_like(a)
```
Then you could just copy paste that here in the doc!